src/torchmetrics/functional/image/lpips.py

```diff
@@ -181,12 +181,12 @@ def forward(self, x: Tensor) -> NamedTuple:
         return vgg_outputs(h_relu1_2, h_relu2_2, h_relu3_3, h_relu4_3, h_relu5_3)
 
 
-def spatial_average(in_tens: Tensor, keepdim: bool = True) -> Tensor:
-    """Spatial averaging over heigh and width of images."""
-    return in_tens.mean([2, 3], keepdim=keepdim)
+def spatial_average(in_tens: Tensor, keep_dim: bool = True) -> Tensor:
```

Comment:
this would be a bc. are you sure?

Reply:
@SkafteNicki can we make this private? :thinking: 
```suggestion
def _spatial_average(in_tens: Tensor, keep_dim: bool = True) -> Tensor:
```